student/models.py

```diff
@@ -8,6 +8,21 @@ class EducationalInstitution(models.Model):
     def __str__(self):
         return self.name
 
+    """
```

Comment:
In python you put the function docstrings within the body of the function in the 1st line - that makes them available via the `help()` builtin function, show up in IDEs, etc.

Reply:
Got it, I'm a bit new to python, I'll change it :)